docs/docs/guides/labs/freshness/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Freshness Policies'
+sidebar_position: 100
+unlisted: True
+---
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+## Overview
+
+Freshness policies help you understand which of your assets have materialized recently and which ones are running behind - a key component of asset health. Freshness policies also communicate expectations for data freshness, allowing downstream asset consumers to determine how often assets are expected to be updated.
+
+For example, freshness policies can help identify stale assets caused by:
+
+- Misconfigured `AutomationCondition`s
+- Runs not being scheduled due to an upstream failure
+- Runs taking longer than expected to complete
+
+### Wait, isn't there already a `FreshnessPolicy`?
+Yes, there is an existing `FreshnessPolicy` API that has been deprecated for quite some time. We're opting to reuse the name for the new freshness APIs, which requires us to migrate off the legacy API.
+
+To avoid naming conflicts with the legacy API during this migration, we've prefixed the new APIs, class names and method args with "internal" (for example, `InternalFreshnessPolicy` and the `internal_freshness_policy` arg on the `@asset` decorator). Once the migration is complete and the new APIs exit preview mode, the "internal" prefix will be removed and `FreshnessPolicy` will be the name of the new APIs.
```

Comment:
```suggestion
To avoid naming conflicts with the legacy API during this migration, we've prefixed the new APIs, class names, and method arguments with "internal", for example, `InternalFreshnessPolicy` and the `internal_freshness_policy` argument on the `@asset` decorator. Once the migration is complete and the new APIs exit preview mode, the "internal" prefix will be removed and `FreshnessPolicy` will be the name of the new APIs.
```